action_server/frontend/src/views/Root.tsx

```diff
@@ -185,7 +185,11 @@ const Root = () => {
               >
                 Runs
               </SideNavigation.Link>
-              <SideNavigation.Link href="/openapi.json" target="_blank" icon={<IconShare />}>
+              <SideNavigation.Link
+                href={serverConfig?.expose_url || '/openapi.json'}
+                target="_blank"
+                icon={<IconShare />}
+              >
```

Comment:
The `href` is updated, but shouldn't the `OpenAPI spec` be changed too here?

Reply:
Added the Public URL to be a separate link, so there would be less confusion:
 
<img width="286" alt="image" src="https://github.com/robocorp/robocorp/assets/31288628/aa2389b5-0809-4d38-b00c-d04a89f76d63">